src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -174,6 +185,11 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
           onBarCodeScanned={onBarCodeScanned}
           onCancel={() => setShouldShowCamera(false)}
           cancelButtonText="Enter ID manually"
+          barCodeTypes={
+            features?.id.scannerType === "CODE_39"
+              ? [BarCodeScanner.Constants.BarCodeType.code39]
+              : [BarCodeScanner.Constants.BarCodeType.qr]
```

Comment:
Hmm I wonder if we should reverse this. Ideally our default should be code39 haha. What do you think haha

Reply:
agreed!